pkgs/by-name/fy/fyi/package.nix

```
@@ -0,0 +1,43 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitea,
+  pkg-config,
+  meson,
+  ninja,
+  dbus,
+  scdoc,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "fyi";
+  version = "1.0.4";
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "dnkl";
+    repo = "fyi";
+    rev = finalAttrs.version;
+    hash = "sha256-UGkShHziREQTkQUlbFXT1144BiBApFVbCvu5A1DuoMI=";
+  };
+
+  depsBuildBuild = [ pkg-config ];
+
+  nativeBuildInputs = [
+    pkg-config
+    meson
+    ninja
+    scdoc
+  ];
+
+  buildInputs = [ dbus ];
+
+  meta = {
+    changelog = "https://codeberg.org/dnkl/fyi/releases/tag/${finalAttrs.version}";
+    description = "Command line utility to create desktop notifications";
+    homepage = "https://codeberg.org/dnkl/fyi";
+    license = [ lib.licenses.mit ];
+    mainProgram = "fyi";
+    maintainers = with lib.maintainers; [ marnym ];
+    platforms = lib.platforms.linux;
```

```suggestion
    platforms = lib.systems.inspect.patterns.isLinux;
```
Nit: you may want to use the new format for declaring platforms here.

---

Why is this pattern better?

---

It is not considered legacy. https://github.com/NixOS/nixpkgs/blob/b03ac42b0734da3e7be9bf8d94433a5195734b19/lib/meta.nix#L75-L81